src/cli/format.js

```diff
@@ -145,10 +145,11 @@ async function format(context, input, opt) {
       /* istanbul ignore next */
       if (ast !== past) {
         const MAX_AST_SIZE = 2097152; // 2MB
-        const astDiff =
-          ast.length > MAX_AST_SIZE || past.length > MAX_AST_SIZE
-            ? "AST diff too large to render"
-            : diff(ast, past);
+        const astDiff = (
+            ast.length > MAX_AST_SIZE || past.length > MAX_AST_SIZE
+          ) ?
+            "AST diff too large to render"
+          : diff(ast, past);
```

Comment:
I think this should instead be:
```js
        const astDiff = 
          ast.length > MAX_AST_SIZE || past.length > MAX_AST_SIZE ?
            "AST diff too large to render"
          : diff(ast, past);
```

Reply:
Not sure why this is happening, since the `test` is basically in its own group, with roughly

```
group([wrapInParens(print("test")), " ?"])
```

Reply:
Just kidding, that would be 82 characters; this is working as expected.
```js
        const astDiff = ast.length > MAX_AST_SIZE || past.length > MAX_AST_SIZE ?
            "AST diff too large to render"
          : diff(ast, past);
```